### PR TITLE
New dues payment reminder email

### LINKED ISSUE #1594: Delay dues form reminder email
From @barulicm - to reduce the email spam we send, we should delay the ACTION REQUIRED email for a few minutes and skip it if payment has been received in the interim

### PR DESCRIPTION
**Pull request by kberzinch**
Closes #1594

### CODE REVIEW

**Comment by kberzinch** [comment]
need to figure out why `ShouldBeUnique` not working

File: resources/views/mail/duespaymentreminder.blade.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+You still need to make a ${{ intval($transaction->package->cost) }} payment for {{ $transaction->package->name }} dues. If you would like to pay for a different term, visit {{ route('showDuesFlow') }}.
+
+You can pay online with a credit or debit card at {{ route('pay.dues') }}. Note that we add an additional ${{ number_format(\App\Models\Payment::calculateSurcharge($transaction->package->cost * 100) / 100, 2) }} surcharge for online payments.
+
+If you would prefer to pay by cash or check, please bring it to an officer or project manager at the shop.
+
+Please write checks to Georgia Tech, with RoboJackets on the memo line. Don't forget to sign it!
+
+If you have any questions, just reply to this email.
```
Comment (kberzinch) [inline resources/views/mail/duespaymentreminder.blade.php:11]: @keenebrogan would you rather direct people to Slack? do you have any other feedback on the language here?

**Review by JoshOldenburg — APPROVED** [review]

File: app/Observers/DuesTransactionObserver.php [hunk 2]
```diff
@@ -6,13 +6,18 @@
...
 class DuesTransactionObserver
 {
     public function saved(DuesTransaction $duesTransaction): void
     {
         CreateDuesPaymentDueNotificationInNova::dispatch($duesTransaction->user);
+
         PruneDuesNotificationsInNova::dispatch($duesTransaction->user);
+
+        SendDuesPaymentReminder::dispatch($duesTransaction->user)
+            ->delay(now()->addHours(48)->hour(10)->startOfHour());
```
Comment (kberzinch) [inline app/Observers/DuesTransactionObserver.php:21]: I think I'll add this to the paymentobserver as well

**Review by evan10s — COMMENTED** [review]
Code looks OK. I think it would be good to write some tests to confirm that notification emails are sent at the right times and quantities.

File: resources/views/mail/duespaymentreminder.blade.php (hunk 1 above)
Comment (keenebrogan) [reply resources/views/mail/duespaymentreminder.blade.php:11]: As long as their questions are directed to me somehow, I don't think it particularly matters. Rest of the message LGTM

**Review by keenebrogan — APPROVED** [review]
Reminder message looks good to me

**Review by JoshOldenburg — APPROVED** [review]

**Review by evan10s — APPROVED** [review]

**Pull request merged**